home-docs/README.md

```diff
@@ -0,0 +1,27 @@
+- [modules-description.md](https://github.com/ita-social-projects/Home/blob/dev/home-docs/modules-description.md) -
+  description and functionality of project modules
+
+
+- [project-description.md](https://github.com/ita-social-projects/Home/blob/dev/home-docs/project-description.md) -
+  project description, used technologies and tools
+
```

Comment:
Maybe it would be better to specify project-description.md first and modules-description.md second?